src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -162,6 +165,9 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeModeOptions, Geocoder
             $.extend(otpOptions, {optimize: 'TRIANGLE'},
                      bikeModeOptions.options.bikeTriangle[bikeTriangle]);
             UserPreferences.setPreference('bikeTriangle', bikeTriangle);
+
+            // allow longer bike riding when using public transit
+            $.extend(otpOptions, { maxWalkDistance: MAXWALK_BIKE * METERS_PER_MILE });
```

Comment:
I'm not sure it should be updated -- just sent with the OTP request. Otherwise when you switch back to walking mode, won't the user selectable preference be overridden with the bike default?